CHANGES.md

```diff
@@ -4,6 +4,8 @@ Changes
 development (master)
 --------------------
 
+- Use named loggers, default `confidence.*` loggers to silence
```

Comment:
From the diff I'm missing the reason for defaulting logging to silence.

Reply:
A good point, let's add something along the lines of the PR description to `CHANGES.md`. Maybe also mention this in the docs, even though those are rather glass-half-empty still.

- [ ] add rationale for silent loggers to changelog
- [ ] add logging section to docs